css/files_sharing.css

```
@@ -0,0 +1,266 @@
+
+	div#files-public-content {
+		.files-controls {
+			height: 80px;
+			padding: 22px 16px;
+			.actions.creatable {
+				position: absolute;
+				right: 16px;
+
+				.button.new {
+					background-color: var(--ion-button-primary-background-default);
+					color: var(--ion-button-primary-text);
+					border: none;
+					border-radius: 30px;
+					padding: 7px 16px;
+					min-height: 36px;
+					height: 36px;
+
+					&:disabled {
+						background-color: var(--ion-button-primary-background-disabled);
+						color: var(--ion-button-primary-text-disabled);
+					}
+
+					&:hover:not(:disabled):not(.button-vue--disabled) {
+						background-color: var(--ion-button-primary-background-hover);
+						border-color: var(--ion-button-primary-background-hover);
+					}
+
+					&:active:not(:disabled):not(.button-vue--disabled) {
+						background-color: var(--ion-button-primary-background-active);
+						border-color: var(--ion-button-primary-background-active);
+					}
+				}
+				.newFileMenu {
+					top: 50px;
+					left: -80px !important;
+					padding: 12px;
+					background-color: var(--ion-color-main-background);
+					border: 1px solid var(--ion-context-menu-border);
+					border-radius: 16px;
+					/* Todo: Place Menu above grid view icon. Input appreciated */
+					z-index: 100000;
+
+					&::after{
+						display: none;
+					}
+
+					ul {
+						gap: 12px;
+					}
+				}
+
+				#uploadprogresswrapper {
+					position: absolute;
+					right: 120px;
+
+					:has([style*="display:none"]) {
+						display: none;
+					}
+				}
+			}
+		}
+
+
+		#imgframe {
+			padding-bottom: 16px;
+
+			&:not(:has(#viewer)) {
+				margin-top: 20%;
+			}
+		}
+
+		#downloadFile.button {
+			margin-top: 48px;
+			background-color: var(--ion-button-secondary-background-default);
+			border: var(--ion-button-secondary-border-default);
+			color: var(--ion-button-secondary-text);
+			border-radius: 50px;
+			font-size: 14px;
+			padding: 8px 16px;
+			min-height: 36px;
+			height: 36px;
+			display: inline-flex;
+			align-items: center;
+			justify-content: center;
+			gap: 6px;
+
+			&:hover {
+				background-color: var(--ion-button-secondary-background-hover);
+				color: var(--ion-button-secondary-text-hover);
+			}
+
+			&:active {
+				background-color: var(--ion-button-secondary-background-active);
+				border-color: var(--ion-button-secondary-background-active);
+				color: var(--ion-button-secondary-text-active)
+			}
+
+			&:disabled {
+				background-color: var(--ion-button-secondary-background-disabled);
+				color: var(--ion-button-secondary-text-disabled);
+			}
+		}
+	}
+
+	.thumbnail {
+		background-image: var(--original-icon-category-files-dark);
+	}
+
+
+	#view-toggle {
+		top: 85px;
+		right: 20px;
+		position: absolute;
+		z-index: 100;
+	}
+
+	.files-filestable.list-container {
+		tr {
+			border-bottom: 1px solid var(--color-border);
+
+			td,th {
+				border-bottom: none;
+			}
+
+			td:last-child {
+				padding-right: 44px;
+			}
```

padding on the last `td` causes that header is not aligned to the column "size" and "modified". Header overlaps the view switch button. Please procede the same way with the table header too, may be like this
```suggestion
			td:last-child {
				padding-right: 44px;
			}
			th:last-child {
				padding-right: 60px;
			}
```